mountpoint-s3/src/inode/readdir.rs

```diff
@@ -233,11 +213,10 @@ impl ReaddirEntry {
         }
     }
 
-    /// How to describe this entry in an error message about shadowing
-    fn shadowed_description(&self) -> String {
+    /// How to describe this entry in an error message
+    fn description(&self) -> String {
```

Comment:
But how would that be different? I think this now just describes the `ReaddirEntry` when mentioning it in a log message, rather than being specifically about shadowing.

Reply:
Hmm, thinking about it again, maybe you're right. The message is general enough so should be fine to use `description`.